packages/fxa-auth-server/lib/subhub/updates.js

```diff
@@ -0,0 +1,73 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+*/
+
+'use strict';
+
+const Joi = require('joi');
+
+const MESSAGE_SCHEMA = Joi.object().keys({
```

Comment:
This should be checked against subhub, but I don't think it's actually generating these messages yet.